requirements.txt

```diff
@@ -3,7 +3,6 @@
 
 beautifulsoup4
 PyYAML==3.10
```

Comment:
I know this is totally unrelated to this PR, but I notice two things about this file:
1. The items are not in alphabetical order. Perhaps we don't try to do that.
2. I don't believe this project needs `PyYAML` anymore.

I guess a separate issue for that would be appropriate.